mappings/net/minecraft/entity/LightningEntity.mapping

```diff
@@ -1,16 +1,16 @@
 CLASS app net/minecraft/entity/LightningEntity
-	FIELD b randomSeed J
-	FIELD c ticksBeforeDespawn I
-	FIELD d fireSpawnCount I
-	FIELD e preventDamage Z
-	FIELD f causingPlayer Luv;
+	FIELD b seed J
+	FIELD c ambientTick I
+	FIELD d remainingActions I
+	FIELD e cosmetic Z
+	FIELD f channeller Luv;
```

Comment:
Okay, those names are an improvement. I'm just a bit curious about the origin of "channeller".

Reply:
This field is set when the LightningEntity is spawned by a channelling trident

Reply:
@InsomniaKitten came up with this name so idk